x-pack/plugins/fleet/server/services/agents/crud.ts

```diff
@@ -315,11 +315,11 @@ export async function bulkUpdateAgents(
   });
 
   return {
-    items: res.body.items.map((item: estypes.BulkResponseItemContainer) => ({
+    // @ts-expect-error ErrorCause is not assignable to Error
+    items: res.body.items.map((item) => ({
```

Comment:
I'm not sure these typings are correct.

Reply:
@nchaulet I believe you're more familiar with this code. Should we be wrapping the `item.update!.error` with `new Error()`?

Reply:
Looking at the code looks like we never consume these errors, I am going to investigate a little more what error handling we are missing, but it should not block that PR.